screens/booking/AllBookings.tsx

```diff
@@ -0,0 +1,142 @@
+import { View, StyleSheet, FlatList } from "react-native";
+
+import CompletedTimeSlot from "../../components/booking/CompletedTimeSlot";
+
+const mock = [
```

Comment:
Should we separate this one in another file?

Reply:
Done. Moved to new file for now.